packages/ui/Button/Button.js

```diff
@@ -156,13 +155,6 @@ const ButtonBase = kind({
 		}
 	},
 
-	handlers: {
```

Comment:
Need to update the docs for `disabled` to indicate that it's up to `Touchable` or the controlling component to prevent clicks from firing.  That for `ButtonBase` it's just styling.